### ISSUE TITLE
Contribution guide?

### ISSUE THREAD

**Issue by AnkushMalaker** [kurema/CitiesSkylinesModsRepoForEpicUsers#3]
Hey, I'd love to contribute to this, particularly, I'd love if there was an attached guide to show how I can compile a binary for a mod myself, and then how I can make a PR to this repo so that others don't have to compile that mod. 

Some way for others to make a request (people can suggest a mod to be compiled and anyone who has time can then compile and make a PR for it) may also be good?

**Comment by kurema** [comment]
That sounds good idea. But there's problems.
First, I'm not that motivated about this repository. Cities: Skylines is just a game I play occasionally, like once in four months (but it can be for a long time).
Second, there's security concern. I don't want to share the mod binary which others say they built. Mod can do lots of malicious thing like crypto mining or data theft. I can't deny the possibility that there is already malicious code on mod repositories published on GitHub and I built and published it without noticing it, but I think it is unlikely. Taking binaries from someone else and publishing it in my repository is another story.

I'm totally fine to add link to another repository. It's easy. That's people's choice to trust or not.

About suggestion, I don't actively receive it. But "Empty It!" is what I added by suggestion in #2. Currently this repository is not popular enough so I'm fine to build it myself one in a while when I feel like it.
